docs/design/proposals/rbd-pv-key-rotation.md

```diff
@@ -0,0 +1,57 @@
+# Encryption Key Rotation
+
+## Proposal
+
+Subject of this proposal is to add support for rotation of encryption keys (KEKs) for encrypted volumes in Ceph-CSI.
+
+## Document Terminology
+
+- Encryption Key: The passphrase that is used to encrypt and open the device.
+- LUKS: The specification used by dm-crypt to process encrypted volumes on linux.
+
+## Proposed Solution
+
+The proposed solution in this document, is to address the periodic rotation of encryption keys for encrypted volumes.
+
+This document outlines the rotation steps for PVCs backed by RBD and will be updated with other volume types as they are supported.
```

Comment:
`will be updated with other volume types as they are supported` what is other volume types here?

Reply:
This doc is for rbd only as the doc name suggest, we don't need to cover cephfs specific things in this doc